src/renderers/shared/stack/event/EventPluginRegistry.js

```diff
@@ -278,9 +278,12 @@ var EventPluginRegistry = {
       if (!dispatchConfig.phasedRegistrationNames.hasOwnProperty(phase)) {
         continue;
       }
-      var pluginModule = EventPluginRegistry.registrationNameModules[
-        dispatchConfig.phasedRegistrationNames[phase]
-      ];
+      let pluginModule;
+      if (dispatchConfig.phasedRegistrationNames) {
```

Comment:
You probably want to extract out this condition outside of the loop. It doesn't make sense to loop on a null value anyway